app/views/static/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<%= javascript_pack_tag 'application' %>
+<%= react_component('App') %>
```

Comment:
- [ ] The view for the root route is created (`root#index`) however It should include only `<div id="root">` The rest should be done with React and Redux.